CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
-## Unreleased
-
+## 0.82.0 / 2025-04-17
+
+* [CHANGE] Add more API validations to the ScrapeConfig CRD. #7413
+* [FEATURE] Add RemoteWrite support for ThanosRuler CRD. #7444 #7455
```

Comment:
removing 7455 since it's only fixing the tests. I'd highlight the stateless mode for clairity.
```suggestion
* [FEATURE] Add RemoteWrite support for ThanosRuler CRD. Note that when remote write is configured, Thanos Ruler runs in stateless mode. #7444
```